PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1072,33 +1084,24 @@ def removePostfix(self, name, postfix):
         return baseName
 
 #====================================================================================================
-    def tuneTxyParameters(self, process, corScheme, postfix):
+    def tuneTxyParameters(self, process, corScheme, postfix, datamc="", campaign="", era="" ):
+        corSchemes = ["Txy", "T1Txy", "T0pcTxy", "T0pcT1Txy", "T1T2Txy", "T0pcT1T2Txy", "T1SmearTxy", "T1T2SmearTxy", "T0pcT1SmearTxy", "T0pcT1T2SmearTxy"]
         import PhysicsTools.PatUtils.patPFMETCorrections_cff as metCors
-        xyTags = {
-            "Txy_50ns":metCors.patMultPhiCorrParams_Txy_50ns,
-            "T1Txy_50ns":metCors.patMultPhiCorrParams_T1Txy_50ns,
-            "T0pcTxy_50ns":metCors.patMultPhiCorrParams_T0pcTxy_50ns,
-            "T0pcT1Txy_50ns":metCors.patMultPhiCorrParams_T0pcT1Txy_50ns,
-            "T1T2Txy_50ns":metCors.patMultPhiCorrParams_T1T2Txy_50ns,
-            "T0pcT1T2Txy_50ns":metCors.patMultPhiCorrParams_T0pcT1T2Txy_50ns,
-            "T1SmearTxy_50ns":metCors.patMultPhiCorrParams_T1SmearTxy_50ns,
-            "T1T2SmearTxy_50ns":metCors.patMultPhiCorrParams_T1T2SmearTxy_50ns,
-            "T0pcT1SmearTxy_50ns":metCors.patMultPhiCorrParams_T0pcT1SmearTxy_50ns,
-            "T0pcT1T2SmearTxy_50ns":metCors.patMultPhiCorrParams_T0pcT1T2SmearTxy_50ns,
-
-            "Txy_25ns":metCors.patMultPhiCorrParams_Txy_25ns,
-            "T1Txy_25ns":metCors.patMultPhiCorrParams_T1Txy_25ns,
-            "T0pcTxy_25ns":metCors.patMultPhiCorrParams_T0pcTxy_25ns,
-            "T0pcT1Txy_25ns":metCors.patMultPhiCorrParams_T0pcT1Txy_25ns,
-            "T1T2Txy_25ns":metCors.patMultPhiCorrParams_T1T2Txy_25ns,
-            "T0pcT1T2Txy_25ns":metCors.patMultPhiCorrParams_T0pcT1T2Txy_25ns,
-            "T1SmearTxy_25ns":metCors.patMultPhiCorrParams_T1SmearTxy_25ns,
-            "T1T2SmearTxy_25ns":metCors.patMultPhiCorrParams_T1T2SmearTxy_25ns,
-            "T0pcT1SmearTxy_25ns":metCors.patMultPhiCorrParams_T0pcT1SmearTxy_25ns,
-            "T0pcT1T2SmearTxy_25ns":metCors.patMultPhiCorrParams_T0pcT1T2SmearTxy_25ns
-            }
-
-        getattr(process, "patPFMetTxyCorr"+postfix).parameters = xyTags[corScheme+"_25ns"]
+        print("DEBUG ",campaign,datamc,era)
```

Comment:
unintentionally left in?

Reply:
Yes, I forgot that line. I just removed it and updated the PR.
Regarding the modifiers, are they also capable of differentiating between 'suberas' e.g. 2018B and 2018C? The corrections also differ between the these suberas.